SQL/user/old_user.sql

```diff
@@ -0,0 +1 @@
+SELECT COUNT(DISTINCT email) FROM users WHERE email = %s;
```

Comment:
Is it possible to put duplicate emails in the database? Would be good to add a unique constraint on email field. Then you wouldn't need `DISTINCT` here.

Reply:
Oh looks like it's already unique. So you can just remove `DISTINCT`.

Reply:
I have lost my SQL dexterity :))) I have to review my notes...